proto/contract.proto

```diff
@@ -1,11 +1,12 @@
 syntax = 'proto3';
 
-import "google/protobuf/empty.proto";
-
 option go_package = "control-plane/pkg/contract";
 option java_package = "dev.knative.eventing.kafka.broker.contract";
 option java_outer_classname = "DataPlaneContract";
 
+// Protobuf include nightmare? No thanks!
+message Empty {}
```

Comment:
Is this safe while upgrading?

Reply:
Can we remove this comment? 
it won't make sense without having context.

Reply:
> Is this safe while upgrading?

I don't think so, even if it's empty, it's still a different message id I guess. But, it's 100% worth it, we might even need to expand this type at some point, so it's more future proof

Reply:
> Can we remove this comment?

:rofl: I may have written this in a moment of rage

Reply:
Now the comment is more specific

Reply:
Thanks! :smile: